prune-releases.sh

```diff
@@ -56,7 +64,7 @@ while [ "$1" != "" ]; do
     shift
 done
 
-if [ -z "$older_than_filter" -o -z "$release_filter" -o -z "$namespace_filter" ]; then
+if [[ -z "$older_than_filter" && -z "$release_filter" && -z "$namespace_filter" && -z "$namespace_negate_filter" && -z "$release_negate_filter" && -z "$max_releases_to_keep" ]]; then
```

Comment:
What does -z do?

Reply:
nvm. Had trouble googling the right words.

Reply:
Hah the minus filter always gets me - I think you can put it in quotes to not make it a minus sign